docs/source/models/risk_effects/hemoglobin/index.rst

```diff
@@ -196,17 +189,21 @@ Relative risk values to calculate custom PAFs for maternal disorders can be acce
                location_id=1, # NOTE: you do not need to specify a location_id. Specifying any location ID will return results specific to location_id=1
                ) 
 
-Use the custom-calculated PAF values such that the maternal disorder incidence rate for an individual :math:`i` for a given affected maternal disorder subcause is as follows:
+Before being applied in the simulation, these relative risk values are first re-scaled to be relative to the TMREL exposure value of 120 g/L. We used linear interpolation to assign RR values for exposures between points provided by the GBD risk curve. For exposure values that fall outside of the range provided by the GBD risk curve, we assign the same RR value as the nearest point on the curve. `See the custom hemoglobin risk effect implementation used in the MNCNH simulation for specific details <https://github.com/ihmeuw/vivarium_gates_mncnh/blob/main/src/vivarium_gates_mncnh/components/hemoglobin.py>`__
 
-.. math::
+We calculate custom PAFs for hemoglobin on maternal disorders outcome for use in the MNCNH simulation. Code to generate these PAFs can be found here ``TODO: POST LINK``. We utilize the interactive context to initialize a population with assigned hemoglobin exposure values, assign relative risk values in accordance with observed exposure, and calculate PAFs among the population as (mean_rr - 1) / mean_rr among the age-stratified population. This PAF calculation process is limited in the following ways:
 
-  ir_i = ir * (1 - PAF) * RR_i
+- The simulated population does not have a uniform distribution of maternal ages. The PAF values for age groups with less person time have less statistical precision than age groups with more person time represented in our simulation.
```

Comment:
Is this effectively saying that we _didn't_ implement the EvenlyDistributedPopulation optimization we did for LBWSG? If so, I feel like this doesn't quite rise to the level of a limitation and maybe should just be a note